lib/Client.ts

```diff
@@ -21,6 +21,7 @@ import type StageChannel from "./structures/StageChannel";
 import Util from "./util/Util";
 
 /* eslint-disable */
```

Comment:
replace this with a set of disables, like you did in shard

Reply:
all done